content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -145,11 +149,19 @@ Run the following commands to set up and deploy Kubeflow.
       '-', and must start and end with an alphanumeric character.
       The value of this variable cannot be greater than 25 characters. It must
       contain just a name, not a directory path.
-      This value also becomes the name of the directory where your Kubeflow
-      configurations are stored, that is, the Kubeflow application directory.
+      You also use this value as directory name when creating the directory where 
+      your Kubeflow  configurations are stored, that is, the Kubeflow application 
+      directory. 
```

Comment:
Suggestion: nit

```suggestion
      You also use this value as a directory name when creating the directory where 
      your Kubeflow configurations are stored—the Kubeflow application 
      directory. 
```

WDYT?

Reply:
I copied this from https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/